EIPS/eip-ice-age-fork.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 2387
+title: "Hardfork Meta: Mountain Glacier"
+author: James Hancock (@madeoftin)
+discussions-to: https://github.com/ethereum/EIPs/pull/2387
+type: Meta
+status: Draft
+created: 2019/11/22
+requires: 1679
+---
+
+## Abstract
+
+This meta-EIP specifies the changes included in the Ethereum hard fork named Mountain Glacier. This hard fork addresses the impending Ice Age on Ethereum Main-net and includes a commitment to solving the problems with the ice age more permanently.
```

Comment:
"Mainnet" is more common in EIPs than "Main-net" (34 vs. 2 eips)

Reply:
updated